lib/src/api.ts

```diff
@@ -2,21 +2,29 @@ import axios from "axios";
 import { Account } from "@ledgerhq/wallet-api-client";
 import BigNumber from "bignumber.js";
 import { decodeSellPayload } from "@ledgerhq/hw-app-exchange";
-import { BEData } from "./sdk";
+import { BEData, ExchangeType } from "./sdk";
 
 const SWAP_BACKEND_URL = "https://swap.ledger.com/v5/swap";
-const SELL_BACKEND_URL = "https://buy.api.aws.prd.ldg-tech.com/sell/v1";
+const SELL_BACKEND_URL = "https://buy.api.aws.prd.ldg-tech.com/";
 
-let axiosClient = axios.create({
+
+let swapAxiosClient = axios.create({
   baseURL: SWAP_BACKEND_URL,
 });
 
+let sellAxiosClient = axios.create({
+  baseURL: SELL_BACKEND_URL,
+});
+
 /**
  * Override the default axios client base url environment (default is production)
  * @param {string} url
  */
 export function setBackendUrl(url: string) {
-  axiosClient = axios.create({
+  swapAxiosClient = axios.create({
+    baseURL: url,
+  });
+  sellAxiosClient = axios.create({
     baseURL: url,
   });
 }
```

Comment:
I guess it works, simple way to manage both backend URL
But I really think it should not matter and the live app creating an exchangeSDK instance **should** always pass the URL needed